src/Administration/Resources/app/administration/src/app/component/structure/sw-page/sw-page.spec.js

```diff
@@ -71,24 +71,6 @@ describe('src/app/component/structure/sw-page', () => {
         expect(wrapper.vm).toBeTruthy();
     });
 
-    it('should use the header bottom-color specified with the headerBorderColor prop', async () => {
-        const wrapper = await createWrapper({
-            meta: {
-                $module: {
-                    color: 'red',
-                },
-            },
-        });
-
-        expect(wrapper.get('.sw-page__head-area').attributes('style')).toBe('border-bottom-color: red; padding-right: 0px;');
-
-        await wrapper.setProps({ headerBorderColor: 'green' });
-
-        expect(wrapper.get('.sw-page__head-area').attributes('style')).toBe(
-            'border-bottom-color: green; padding-right: 0px;',
-        );
-    });
-
```

Comment:
The new Smart Bar design no longer has this colored border. That's why I removed this test.